ts/parts/Utilities.ts

```diff
@@ -64,6 +64,7 @@ declare global {
             complete?: Function;
             curAnim?: Dictionary<boolean>;
             duration?: number;
+            defer?: number;
```

Comment:
Interface properties should be sorted alphabetically (move `defer` above `duration`)